### PR TITLE
Add compatibility note about `Regexp::new` (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: https://github.com/ruby/www.ruby-lang.org/issues/2818

Follow up #3175

Translation diff is a6904d6db074e19fb05b9d7d310b746eff923727

### CODE REVIEW

File: ko/news/_posts/2023-12-25-ruby-3-3-0-released.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ IRB에 여러 개선 사항이 추가됩니다. 다음과 같은 내용이 포
   Ruby 3.4부터 `it`은 첫 번째 블록 인수를 가리킵니다.
   [[Feature #18980]](https://bugs.ruby-lang.org/issues/18980)
 
+* `Regexp::new`는 최대 3개의 인수를 받았지만 이제 2개까지 허용합니다. 이는 Ruby 3.2 에서 폐기예정이었습니다. [[Bug #18797]](https://bugs.ruby-lang.org/issues/18797)
```
Comment (yous) [inline ko/news/_posts/2023-12-25-ruby-3-3-0-released.md:142]:
```suggestion
* `Regexp::new`는 최대 3개의 인수를 받았지만 이제 2개까지 허용합니다. 이는 Ruby 3.2에서 폐기예정이었습니다. [[Bug #18797]](https://bugs.ruby-lang.org/issues/18797)
```

Comment (riseshia) [reply ko/news/_posts/2023-12-25-ruby-3-3-0-released.md:142]: oh

Comment (riseshia) [reply ko/news/_posts/2023-12-25-ruby-3-3-0-released.md]: [c8f654f](https://github.com/ruby/www.ruby-lang.org/pull/3229/commits/c8f654f176b188873e6cf9e41b724f75e8b99331) fixed

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks~

**Pull request merged**